### PR TITLE
Nav 48 user model

### PR DESCRIPTION
**Pull request by tomeksabala**
1. Enabled flask-migration to handle sqlalchemy db initalization and future upgrades
2. Created workflow model to store user state of work with a dataset
3. Added a very simple README for the project
3. A bit of refactors while doing the above work

### CODE REVIEW

File: navigator_api/migrations/versions/9a0a2eabc879_init.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('workflow',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('decision_engine_id', sa.String(), nullable=False),
+    sa.Column('dataset_id', sa.String(), nullable=False),
+    sa.Column('user_id', sa.String(), nullable=False),
+    sa.Column('_skipped_tasks', sa.String(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
```
Comment (jonathansberry) [inline navigator_api/migrations/versions/9a0a2eabc879_init.py:27]: Indentation?

File: navigator_api/migrations/alembic.ini [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# A generic, single database configuration.
+
+[alembic]
```
Comment (jonathansberry) [inline navigator_api/migrations/alembic.ini:3]: Is alembic something that is important for me to get my head round for engine in the next 5 days? 

Given that, engine's entire data structure at this time is actually managed and maintained in Google sheets, and then loaded in a single step into the db, I'm hoping this is something we can push back for now?  i.e. for now we just wipe the db clean and reload everything when we need to.

**Review by jonathansberry — COMMENTED** [review]
The alembic stuff looks pretty cool and a sizeable chunk of work. Just left a couple of comments, but otherwise I understand what is happening here I think.

File: navigator_api/migrations/versions/9a0a2eabc879_init.py (hunk 1 above)
Comment (tomeksabala) [reply navigator_api/migrations/versions/9a0a2eabc879_init.py:27]: This is autogenerated file.

File: navigator_api/migrations/alembic.ini (hunk 2 above)
Comment (tomeksabala) [reply navigator_api/migrations/alembic.ini:3]: That defaul config for alembic used by flask-migration. That's for nav_api db will needs to be persistent and support migrations if we wish to change the db schema in the future.
I think it's alright to go with this vanillia config for this year.

**Pull request merged**